CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```bash
+$ git clone git@github.com:<username>/cryostatio.github.io.git
+$ git remote add upstream git@github.com:cryostatio/cryostatio.github.io.git # Adding upstream as remote
+```
+
+Checkout another branch.
+```bash
+$ cd cryostatio.github.io && git checkout -b some-task-branch
+```
+
+Edit some files and commits your changes.
+```bash
+$ git add . # Stage your changes. Here, we stage the entire repo (recursively).
+$ git commit -m "some very nice message"
```

Comment:
this message seems to become the title- maybe add title guidelines here? (A title following [guidelines](https://www.conventionalcommits.org/en/v1.0.0/).)

Reply:
Right! I think eventually you can edit the title but it makes sense to suggest it here.